pkg/minishift/registration/redhat.go

```diff
@@ -57,6 +63,7 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 		return err
 	}
 	if strings.Contains(output, "not registered") {
```

Comment:
Why not include the first call to the subscription manager in the timing? Maybe we need multiple timings here, one per SSH Command which we just log and cab be used to debugging and one overall timing for the full registration.